datahub/wesim.py

```diff
@@ -112,7 +113,7 @@ def structure_capacity(df: pd.DataFrame) -> pd.DataFrame:
     return df.reset_index().replace({"Code": REGIONS_KEY})
 
 
-def get_wesim() -> dict[str, pd.DataFrame]:
+def get_wesim() -> dict[str, Any]:  # type: ignore[misc]
```

Comment:
This should probably be dict since each element contains a dict
```suggestion
def get_wesim() -> dict[str, dict]:
```